_docs/concepts/what-is-istio/goals.md

```diff
@@ -8,7 +8,9 @@ layout: docs
 type: markdown
 ---
 
-This page the core principles that guide Istio's design.
+This page outlines the core principles that guide Istio's design and explains some key design decisions based on these goals.
+
+## Design goals
```

Comment:
There's already an H1 block that says Design Goals that's inserted right above, this seems redundant.